napari/layers/points/points.py

```diff
@@ -471,6 +540,45 @@ def properties(self, properties: Dict[str, np.ndarray]):
             self._face_color_property = ''
             warnings.warn('property used for face_color dropped')
 
+        if self._edge_color_property and (
+            self._edge_color_property not in self._properties
+        ):
+            self._edge_color_property = ''
+            warnings.warn('property used for edge_color dropped')
+
+    @property
+    def current_properties(self):
+        """Dict[str: np.ndarray]: properties for the next added point."""
```

Comment:
The type annotation should be a type annotation rather than part of the docstring. =)